src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt

```diff
@@ -43,7 +45,7 @@ class ChristmasEventPlugin : JavaPlugin() {
     var canJoin = false
 
     companion object {
-        val instance: ChristmasEventPlugin by lazy { getPlugin(ChristmasEventPlugin::class.java) }
+        val INSTANCE: ChristmasEventPlugin by lazy { getPlugin(ChristmasEventPlugin::class.java) }
```

Comment:
Technically, a `by lazy` property is not a compile-time constant, so even if the variable is to emphasise its role as a global, immutable, always-present reference, idiomatically it is correct to use camelCase.